test/eipmi_response_tests.erl

```diff
@@ -253,6 +253,103 @@ decode_get_sdr_test() ->
              {data, <<$d, $a, $t, $a>>}]},
        eipmi_response:decode(Resp, Bin)).
 
+decode_get_chassis_caps_test() ->
+    Resp = {?IPMI_NETFN_CHASSIS_RESPONSE, ?GET_CHASSIS_CAPABILITIES},
+    Bin = <<15:8, 27:8, 132:8, 65:8, 213:8, 40:8>>,
+    ?assertEqual(
+       {ok, [{interlock, 1},
```

Comment:
It would be cool to have the returned types equal the setter types, e.g. currently `interlock` is of type `boolean()` when you set the caps, but of type `0|1` when you get them.